synapse/handlers/room_member.py

```diff
@@ -491,6 +491,8 @@ def _update_membership(
                 else:
                     # send the rejection to the inviter's HS.
                     remote_room_hosts = remote_room_hosts + [inviter.domain]
+                    # Note: This is calling _remote_reject_invite on RoomMemberMasterHandler,
```

Comment:
>  This is calling _remote_reject_invite on RoomMemberMasterHandler

well, that or RoomMemberWorkerHandler, which also has a "requester" arg.

The problem here seems to be simply that the abstract class definition (`RoomMemberHandler`) is incorrect. I suggest you fix that rather than adding this comment.

Reply:
Y'know I never even considered that a possibility.